src/packages/console/src/App.tsx

```diff
@@ -105,6 +105,9 @@ const theme = buildTheme({
     error: {
       color: '$colors.error',
     },
+    selectItemText: {
+      'font-size': '14px',
```

Comment:
font sizes should be in em

Reply:
please check the lint error, it has something to do with the everyview-tracker